mountpoint-s3/scripts/fs_latency_bench.sh

```diff
@@ -127,8 +127,9 @@ do
 done
 
 
-# start time to first byte benchmark
-jobs_dir=mountpoint-s3/scripts/fio/read_latency
+run_start_time_to_first_byte_benchmarks() {
```

Comment:
We probably should change this function name too. It's not time to first byte for the write benchmark.